src/plugins/irc-events/unhandled.js

```diff
@@ -14,6 +15,7 @@ module.exports = function(irc, network) {
 		network.channels[0].pushMessage(client, new Msg({
 			type: Msg.Type.UNHANDLED,
 			command: command.command,
+			commandslug: slugify(command.command),
```

Comment:
Makes sense, this just seemed like the easiest place to throw it. You could do it using a replace, I'm just paranoid about any dodgy messages coming through from the server or commands using weird characters, which is why I'd tend towards slugifying it fully I guess.